glib is not properly linked, changing the variable so it is present
--- src/src/CMakeLists.txt	2021-04-13 18:13:27.000000000 +0000
+++ src.mod/src/CMakeLists.txt	2021-05-20 14:03:49.449914000 +0000
@@ -75,7 +75,7 @@
 target_link_libraries(pavucontrol-qt
     Qt5::Widgets
     ${PULSE_LDFLAGS}
-    ${GLIB_LDFLAGS}
+    ${GLIB_LIBRARIES}
 )
 
 install(TARGETS
